src/AudioToolbox/AudioSession.cs

```diff
@@ -168,10 +171,13 @@ NSArray Extract (IntPtr key, NSString secondKey)
 			}
 		}
 
-#if !NET
-		[Deprecated (PlatformName.iOS, 7, 0)]
-#else
+#if NET
 		[UnsupportedOSPlatform ("ios7.0")]
+#if IOS
+		[Obsolete ("Starting with ios7.0.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```

Comment:
@chamons  we are using ""Starting with macos10.14 not available anymore."" I think you have an issue with that message, am I wrong?

Reply:
I think this is fine, I mean it is `obsolete` - `starting with iOS7.0.` also if people need more context we already include the `UrlFormat` pointing to further explanation

Reply:
On the other hand I just realized what Manuel means so yeah for consistency with the other PRs -> `Starting with ios7.0 not available anymore.`

Reply:
I think the confusion is coming from this one https://github.com/xamarin/xamarin-macios/pull/13791#discussion_r788221530 so here the tool is doing the right thing and nothing needs to be changed.

Reply:
The error you get from dotnet build looks like:

/Users/donblas/tmp/obsolete/Program.cs(4,1): warning BI1234: 'Foo.Bar()' is obsolete: 'Starting with ios7.0.' [/Users/donblas/tmp/obsolete/obsolete.csproj]

So I'm pretty happy with that. Thoughts @mandel-macaque ?